document_sftp/hooks.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import logging
+import StringIO
+import socket
+from lxml import etree
+from paramiko.ecdsakey import ECDSAKey
+from openerp import SUPERUSER_ID, tools
+_logger = logging.getLogger(__name__)
+
+
+def post_init_hook(cr, pool):
+    if socket.getfqdn().endswith('odoo-community.org'):  # pragma: no cover
+        # we need a different default port on runbot
```

Comment:
maybe you could use the port 0 trick? You will need to log the resulting port (and maybe set a system parameter) so that ppl get to know which port is used.